### PR TITLE
Fix Item subject

### PR DESCRIPTION
**Pull request by serious6**
Subject should be a String. Also added a simple test

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@serious6__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by serious6** [comment]
fixes #388

File: src/test/java/microsoft/exchange/webservices/data/core/service/items/AppointmentTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  @Test
+  public void testSetSubject() throws Exception {
+    doReturn(ExchangeVersion.Exchange2010_SP2).when(exchangeService).getRequestedServerVersion();
+
+    Appointment appointment = new Appointment(exchangeService);
+
+    final String subject = "Lorem Ipsum";
+    appointment.setSubject(subject);
+
+    assertThat(appointment.getSubject(), allOf(not(isEmptyOrNullString()), equalTo(subject)));
```
Comment (vboctor) [inline src/test/java/microsoft/exchange/webservices/data/core/service/items/AppointmentTest.java]: Do we really need the null check?  Why not just that it is equal to subject?

**Comment by vboctor** [comment]
:+1: I assume you will be cherry-picking this to master as well.

File: src/test/java/microsoft/exchange/webservices/data/core/service/items/AppointmentTest.java (hunk 1 above)
Comment (serious6) [inline src/test/java/microsoft/exchange/webservices/data/core/service/items/AppointmentTest.java]: yes should be obsolet. I will remove it.

**Comment by serious6** [comment]
yes. That s the plan. Do you have any comments/suggestions to this behavior?

**Pull request merged**